.htaccess

```diff
@@ -119,6 +122,7 @@
 	RewriteRule ^(.*)/topics/authentication$ /$1/developer_guides/security/authentication [R=301,L]
 	RewriteRule ^(.*)/topics/caching$ /$1/developer_guides/performance/caching [R=301,L]
 	RewriteRule ^(.*)/topics/commandline$ /$1/developer_guides/cli [R=301,L]
+	RewriteRule ^/topics/datamodel/?$ /en/developer_guides/model/data_model_and_orm  [R=301,L]
```

Comment:
Or, use ^([a-z]{2}) as on line 57/59? Do you have a preference ?

Question: how do we know whether or not we are adding rewrites on top of existing rewrites ? Might it be better to comment all these rewrite rules out, and start from scratch with a new top 50 of the 404s ?